src/runtime_src/core/pcie/tools/xbmgmt/cmd_part.cpp

```diff
@@ -118,7 +118,7 @@ int program_prp(unsigned index, const std::string& xclbin, bool force)
 
         int pending = 1;
         int retry = 0;
-        while (pending && retry < PROGRAM_TIMEOUT) {
+        while (pending && retry++ < PROGRAM_TIMEOUT) {
```

Comment:
To prevent "future" developer interpretation issues, could you change the while loop to:
`while (pending && (retry++ < PROGRAM_TIMEOUT)) {`